.github/workflows/update-images.yml

```diff
@@ -14,11 +18,13 @@ jobs:
           # Latest branches
           - { branch: master, channel: latest/edge }
           # Stable branches
+          - { branch: "1.31", channel: "1.31" }
```

Comment:
I guess we should also add 1.32?

Reply:
Yes, once we have release branches.

Reply:
Ah, my mind was stuck in the k8s-snap universe. Ignore my comment...